fix(build)!: Upgrade to Grails 6

matrei opened the pull request:
- New Java 11 baseline
- Create new project layout, updated for Grails 6/Gradle 7.6
- Replace Travis with Github actions
- Add release drafter
- Add Renovate settings

matrei committed:
fix(build)!: Upgrade to Grails 6

- New Java 11 baseline
- Create new project layout, updated for Grails 6/Gradle 7.6
- Replace Travis with Github actions
- Add release drafter
- Add Renovate settings

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grails/grails-quartz?pullRequest=125) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grails/grails-quartz?pullRequest=125) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grails/grails-quartz?pullRequest=125) it.</sub>

matrei committed:
chore(): Remove travis files

matrei committed:
fix(readme): Update readme

matrei committed:
chore(): Update links

matrei committed:
fix(config): Remove autoStartup from plugin.yml

autoStartup should not have been added here.
It should default to false and does so in QuartzGrailsPlugin.

matrei committed:
fix(build): Add Gradle settings

matrei committed:
fix(ci): Set correct sonatype publish properties

fernando88to commented:
It's great to see this project being updated, congratulations.

One suggestion is to replace the 'grails create-job" command with a GrailsApplicationCommand, for example ./gradlew runCommand -Pargs="create-job MyJob"

matrei commented:
> One suggestion is to replace the 'grails create-job" command with a GrailsApplicationCommand, for example ./gradlew runCommand -Pargs="create-job MyJob"

@fernando88to I'm not sure I follow, `create-job` is a "first-class" Grails CLI command again since https://github.com/grails/grails-forge/pull/276
